packages/expo-av/CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ _This version does not introduce any user-facing changes._
 
 - Remove peerDependencies and unimodulePeerDependencies from Expo modules. ([#11980](https://github.com/expo/expo/pull/11980) by [@brentvatne](https://github.com/brentvatne))
 
+- Fixed the web Video Fullscreen APIs in Safari ([#12258](https://github.com/expo/expo/pull/12258) by [@elliotdickison](https://github.com/elliotdickison))
+
```

Comment:
This entry is in the wrong place. It should be under `## Unpublished -> ### 🐛 Bug fixes`

Reply:
doh! fixed